osc/obs_api/choices.py

```diff
@@ -0,0 +1,78 @@
+BLOCK_MODES: tuple[str] = (
```

Comment:
Note that `tuple[str]` is a relatively recent syntax, might be Python 3.9+ or 3.10+. Also, I'd agree with @mimi1vx that these should be `StrEnum` instead.